src/Workspaces/Core/Portable/FindSymbols/SymbolTree/SymbolTreeInfo_Metadata.cs

```diff
@@ -13,6 +14,18 @@ namespace Microsoft.CodeAnalysis.FindSymbols
 {
     internal partial class SymbolTreeInfo
     {
+        private static Metadata GetMetadataNoThrow(PortableExecutableReference reference)
+        {
+            try
+            {
+                return reference.GetMetadata();
```

Comment:
No, that wouldn't handle BadImageFormatException. The method documents exceptions that need to be handled.